introductions/aidancole.html

```diff
@@ -0,0 +1,110 @@
+<!DOCTYPE html>
+
+<html>
+<head>
+  <meta charset="utf-8">
+
+  <!-- 0. Update the title below. -->
+  <title>Introduction: Aidan Cole</title>
+
+  <link href="https://cdn.jsdelivr.net/npm/bootstrap@5.1.3/dist/css/bootstrap.min.css" rel="stylesheet" integrity="sha384-1BmE4kWBq78iYhFldvKuhfTAU6auU8tT94WrHftjDbrCEXSU1oBoqyl2QvZ6jIW3" crossorigin="anonymous">
+  <link rel="stylesheet" href="https://unpkg.com/leaflet@1.7.1/dist/leaflet.css" integrity="sha512-xodZBNTC5n17Xt2atTPuE1HxjVMSvLVW9ocqUKLsCC5CXdbqCmblAshOMAS6/keqq/sMZMZ19scR4PsZChSR7A==" crossorigin=""/>
+  <link href="styles/introductions.css" rel="stylesheet">
+</head>
+
+<body>
+  <header id="identity">
+    <!-- 1. Create an h1 element containing your preferred name. -->
+  <h1>Aidan Patrick Cole<h1>
+    <!-- 2. (optional) Create a paragraph (p) element, set the class
+            attribute to "pronouns", and add your preferred pronouns inside of
+            the element. -->
+    <div class="pronouns">
+      <p>He/him<p>
+    <!-- 3. Fill in the links to your GitHub and Slack profiles below. -->
+    <ul class="profiles">
+      <li class="github-profile"><a href="https://github.com/aidanpcole"></a></li>
+      <li class="slack-profile"><a href="https://musa6112022.slack.com/team/U030L776RDJ"></a></li>
```

Comment:
@aidanpcole The reason these two were failing the tests is because there should be some label between the `<a>...</a>` tags. The `href` attribute tells the browser where the link should take the user when clicked, but the text that comes between the tags determines what's actually shown on the page for the user to click on.